src/test/java/org/commcare/formplayer/utils/GenerateSnapshotDatabases.java

```diff
@@ -63,8 +83,13 @@ public void testCreateSnapshot() throws Exception {
             if (new File(SQLiteProperties.getDataDir()).exists()) {
                 fail("Couldn't remove existing snapshot assets");
             }
-            syncDb();
-            doInstall("sandbox_reference/snapshot.json");
+            new SyncDbRequest(mockMvc, restoreFactory).request();
```

Comment:
we should probably delete `syncDb()` from `BaseTestClass` and replace it with the new `SyncDbRequest`